### PR TITLE
Add biomedical transformers review

### PR DESCRIPTION
**Pull request by nsorros**
Adds a summary of the literature review I did. The main points I am adding have to do with things relevant to the project. Happy to add more information in case you think they are useful for others though.

### CODE REVIEW

**Comment by nsorros** [comment]
[Literature review of transformers improvements that applies to mesh](https://trello.com/c/GwnFnLOV/1155-literature-review-of-transformers-improvements-that-applies-to-mesh)

File: docs/literature_review.md [hunk 1]
```diff
@@ -1,6 +1,84 @@
...
+
+# Small practical BERT
+https://arxiv.org/pdf/1909.00100.pdf
+
+![distill](https://user-images.githubusercontent.com/4975761/104293533-c7f65700-54c6-11eb-94ec-3d375f3971d5.png)
+
+Distillation is more effective than pre training from scratch.
+
+
+General review
```
Comment (lizgzil) [inline docs/literature_review.md:81]: is this a sub title?

File: docs/literature_review.md [hunk 2]
```diff
@@ -1,6 +1,84 @@
...
 
-Overview
+Transformers review
+
+* Pre training on domain data from scratch is the best approach
+* A more computation efficient approach, if your dataset is small, is to create an expanded data from bringing the closest examples from the domain and pre training there
+* Training from scratch a smaller BERT performs much worse than distilling a bigger BERT into a smaller one
+
+# Don't stop pre training
+https://www.aclweb.org/anthology/2020.acl-main.740.pdf
```
Comment (lizgzil) [inline docs/literature_review.md:10]: this is a personal thing, so feel free to ignore - but I really like to see the names of the authors and years at a glance in a literature review. So would prefer those things with a hyperlink than just the url here.

**Review by lizgzil — APPROVED** [review]
looks good - but a general point about structure - could you separate out the optimising transformers based models from the general transformers review?

File: docs/literature_review.md (hunk 1 above)
Comment (nsorros) [reply docs/literature_review.md:81]: it should be, good point

File: docs/literature_review.md (hunk 2 above)
Comment (nsorros) [reply docs/literature_review.md:10]: ok so transform that to a link and add First author et al, Year?

Comment (lizgzil) [reply docs/literature_review.md:10]: yeh, thatd be my preference anyway

File: docs/literature_review.md [hunk 3]
```diff
@@ -1,6 +1,87 @@
...
+which is defined as pre training on the same data distribution as the data of
+the task i.e. unlabelled examples, but their combination performs slightly
+better than just domain adaptive pre training.
+
+One of the reasons DAPT outperforms is that the are usually much more data
+in the domain than the task.
+
+![curated tapt](https://user-images.githubusercontent.com/4975761/104292923-0d665480-54c6-11eb-8062-1db6652097b3.png)
+
+You can approach DAPT performance by selecting a subset of the domain data using
```
Comment (aCampello) [inline docs/literature_review.md:26]: That's very interesting. Under which metric? Cosine similarity?

File: docs/literature_review.md [hunk 4]
```diff
@@ -1,6 +1,87 @@
...
+
+![dapt and tapt](https://user-images.githubusercontent.com/4975761/104292786-ed369580-54c5-11eb-98c1-f67fef49ef0c.png)
+
+Domain adaptive pre training (DAPT), which is defined as pre trained on
+generic domain data like biomedical, is better than task adaptive pre training,
+which is defined as pre training on the same data distribution as the data of
+the task i.e. unlabelled examples, but their combination performs slightly
+better than just domain adaptive pre training.
+
+One of the reasons DAPT outperforms is that the are usually much more data
```
Comment (aCampello) [inline docs/literature_review.md:21]: That makes sense.

File: docs/literature_review.md (hunk 3 above)
Comment (nsorros) [reply docs/literature_review.md:26]: Good point, i should have specified. Using some vectorisation like tfidf and then some fast way to calculate similarities

**Pull request merged**